### PR TITLE
added release notes from the last sprint

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-2024-0/devices-2024-0.md [hunk 1]
```diff
@@ -109,20 +109,44 @@ Build version</th>
...
+<td>Device management</td>
 <td>SmartREST Inventory GET templates created in the UI did not generate responses when there was no external ID type declared in the template. This issue has been addressed for both existing and newly created templates.</td>
 <td>DM-2126</td>
 <td>10.18.7.0</td>
 <td>Core</td>
 </tr>
 
+<tr>
+<td>LWM2M</td>
+<td>The configuration flag <code>fwResetStateMachineOnStart</code> has been added to control if the LWM2M agent resets the firmware update state machine on the client at the beginning of a firmware update. The default of this flag is true which matches the existing behaviour of the LWM2M agent. It is available in the [device registration settings](https://cumulocity.com/guides/protocol-integration/lwm2m/#device-registration-settings).</td>
```
Comment (rssag) [inline content/release-2024-0/devices-2024-0.md:144]: "true" is the value, we sometimes add monospace markup for that. not sure if we do that in release notes

File: content/release-2024-0/platform-2024-0.md [hunk 2]
```diff
@@ -75,7 +75,16 @@ For applications with versions, by default the application with the "latest" tag
...
+<td>The OpenSSL command for generating a "signed verification code" for Proof of Possession has been enhanced to include additional encoding functionality.</td>
+<td>MTM-52775</td>
+<td>10.18.108.0</td>
+<td>UI</td>
+</tr>
+
+
+<tr>
+<td>Authentication</td>
+<td>In the single-sign on configuration page, users can now enable and configure the external token validation process.</td>
```
Comment (rssag) [inline content/release-2024-0/platform-2024-0.md:87]:
```suggestion
<td>In the single-sign-on configuration page, users can now enable and configure the external token validation process.</td>

```

File: content/release-2024-0/devices-2024-0.md (hunk 1 above)
Comment (BeateRixen) [inline content/release-2024-0/devices-2024-0.md:144]:
```suggestion
<td>The configuration flag <code>fwResetStateMachineOnStart</code> has been added to control if the LWM2M agent resets the firmware update state machine on the client at the beginning of a firmware update. The default of this flag is <code>true</code> which matches the existing behaviour of the LWM2M agent. It is available in the [device registration settings](https://cumulocity.com/guides/protocol-integration/lwm2m/#device-registration-settings).</td>
```

Comment (BeateRixen) [reply content/release-2024-0/devices-2024-0.md]: changed

**Pull request merged**